documentation/api/query/examples-pql.markdown

```diff
@@ -0,0 +1,447 @@
+---
+title: "PuppetDB 4.2 » Puppet query language (PQL) » Examples"
+layout: default
+---
+
+[tutorial]: ./tutorial-pql.html
+[reference]: ./v4/pql.html
+
+## Example Queries
+
+Below is a list of example PQL queries that you may find useful.
+
+Other resources you may also find useful include:
+
+* [PQL tutorial][tutorial]
+* [PQL reference guide][reference]
+
+***
+
+### Filtering on node names
+
+Query nodes with `green` in their name.
+
+``` ruby
+nodes { certname ~ 'green' }
+```
+
+*Output:*
+
+``` json
+[
+    {
+        "cached_catalog_status": "not_used",
+        "catalog_environment": "production",
+        "catalog_timestamp": "2016-08-15T11:06:26.275Z",
+        "certname": "greenserver.vm",
+        "deactivated": null,
+        "expired": null,
+        "facts_environment": "production",
+        "facts_timestamp": "2016-08-15T11:06:26.140Z",
+        "latest_report_corrective_change": null,
+        "latest_report_hash": "4a956674b016d95a7b77c99513ba26e4a744f8d1",
+        "latest_report_noop": false,
+        "latest_report_noop_pending": null,
+        "latest_report_status": "changed",
+        "report_environment": "production",
+        "report_timestamp": "2016-08-15T11:06:18.393Z"
+    }
+]
+```
+
+***
+
+### Basic fact filtering
+
+Nodes with operating system name `CentOS`.
+
+``` ruby
+inventory { facts.os.name = "CentOS" }
+```
+
+*Output (abbreviated):*
+
+``` json
+[
+    {
+        "certname": "centos70.vm",
+        "environment": "production",
+        "facts": {
+            ...snip...
```

Comment:
spoke with the docs team about how this would look -- recommendation was to use ... or just leave the snip out, since we don't to that anywhere else.